.dockerignore

```diff
@@ -0,0 +1,5 @@
+.git
```

Comment:
this file shouldn't be checked in?

Reply:
The `.dockerignore` file is functionally equivalent to `.gitignore` except it is used when building docker containers.

Many projects will use the same contents for `.gitignore` and `.dockerignore`, either by copy-paste or file store sym-link [which doesn't work too well on Windows though].

This initial copy of this file was the bare-minimum I could think of to help cut down the amount of files that docker would sucks in when building the containers locally or on build machines, and keeping the build context lean, although maybe other files/directories could be added later.

There are a number of reasons to have a `.dockerignore` file to prevent unintended info leaking in to the docker containers, and this article outlines most of these reasons. 
https://codefresh.io/docker-tutorial/not-ignore-dockerignore/

By checking in a common copy of the `.dockerignore` file, that helps everyone building docker images from this repo, and also should help any PR builds which create docker containers to run a little faster too.